### DISCUSSION TITLE
Small vertical scrollbar shown on Mac 13" Chrome with Bookmarks bar shown

### DISCUSSION THREAD

**Discussion by web-padawan** [elk-zone/elk#1588] [General]
Just started to use Elk and it's awesome! 💯 Here's one minor thing that I immediately noticed though:

| No bookmarks bar         | Bookmarks bar shown  |
|------------------------|------------------------|
| <img width="425" alt="Screenshot 2023-02-03 at 10 02 45" src="https://user-images.githubusercontent.com/10589913/216545388-a39f2f09-7428-4199-9e7f-d8bf501cb197.png"> | <img width="422" alt="Screenshot 2023-02-03 at 10 02 33" src="https://user-images.githubusercontent.com/10589913/216545377-c0eb8408-a149-41cf-b955-f69c1a42524f.png"> |

Would it be a good idea to tweak default styles a little bit, so that there is no vertical scrollbar in this case?

**Comment by ZMYaro** [comment]
There is a broader discussion of this in #894.

**Reply by web-padawan** [reply]
Thanks. Sorry about a duplicate, I couldn't quickly find that discussion.
